api/queries_repo.go

```diff
@@ -863,7 +863,8 @@ type RepoMetadataInput struct {
 	Assignees  bool
 	Reviewers  bool
 	Labels     bool
-	Projects   bool
+	ProjectsV1 bool
```

Comment:
I've introduced this new field because it is important to me that we make the decision up further up the stack instead of prop drilling the support variable through. It was also a real smell that this function accepted one input for both but split the result by v1 v2